### PR TITLE
adding git+ssh uri scheme

### PR DESCRIPTION
**Pull request by vmindru**
an obvious example of git+ssh would have saved me today 5 minutes of googling :-D

### CODE REVIEW

File: docs/docsite/rst/galaxy.rst [hunk 1]
```diff
@@ -134,6 +134,10 @@ Use the following example as a guide for specifying roles in *requirements.yml*:
       scm: git
       version: "0.1"  # quoted, so YAML doesn't parse this as a floating-point value
 
+    # using git+ssh:// URI scheme
+    - src: git+ssh://git@github.com:vmindru/ansible.git
```
Comment (sivel) [inline docs/docsite/rst/galaxy.rst]: This format is no different that the one above referenced as `from GitLab or other git-based scm` except that it adds the scheme which is redundant.

The `git+ssh://` scheme is inferred when using an scp-like syntax.

Comment (vmindru) [reply docs/docsite/rst/galaxy.rst]: hi, tahnks for the comment. can you be more specific about "scp-like" syntax?

Comment (sivel) [reply docs/docsite/rst/galaxy.rst]: Please see https://git-scm.com/book/en/v2/Git-on-the-Server-The-Protocols#_the_ssh_protocol

**Comment by acozine** [comment]
@vmindru thank you for taking the time to open this PR. I've incorporated the spirit of your suggestion into the galaxy documentation by adding a note about `git+ssh` to the example that worked for your use case.

**Pull request merged**